aio/content/examples/what-is-angular/src/app/hello-world/hello-world.component.ts

```diff
@@ -0,0 +1,13 @@
+// #docplaster
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'hello-world',
+  template: `<h2>Hello World</h2>
+                <p>This is my first component!</p>
+             `,
```

Comment:
```suggestion
  template: `
    <h2>Hello World</h2>
    <p>This is my first component!</p>`,
```
I just noticed the `<p>` was indented too much (it should be at the same level of indent as the `h2`